src/Agent.Worker/Build/GitSourceProvider.cs

```diff
@@ -1072,13 +1075,16 @@ public override async Task RunMaintenanceOperations(IExecutionContext executionC
 
             // Initialize git command manager
             _gitCommandManager = HostContext.GetService<IGitCommandManager>();
-#if OS_WINDOWS
-            // On Windows, we will always find git from externals
-            await _gitCommandManager.LoadGitExecutionInfo(executionContext, useBuiltInGit: true);
-#else
-            // On linux/OSX, we will always find git from %PATH%
-            await _gitCommandManager.LoadGitExecutionInfo(executionContext, useBuiltInGit: false);
-#endif
+            if (PlatformUtil.RunningOnWindows)
+            {
+                // On Windows, we will always find git from externals
+                await _gitCommandManager.LoadGitExecutionInfo(executionContext, useBuiltInGit: true);
```

Comment:
NIT: Would prefer:

```
bool useBuiltInGit = PlatformUtil.RunningOnWindows;
 await _gitCommandManager.LoadGitExecutionInfo(executionContext, useBuiltInGit: useBuiltInGit );
```

instead of if/else